packages/transformer/src/IModelExporter.ts

```diff
@@ -158,9 +158,6 @@ export abstract class IModelExportHandler {
     return true;
   }
 
-  /** Called when element is skipped instead of exported. */
-  public onSkipElement(_elementId: Id64String): void {}
```

Comment:
Another reason was that it could be misleading. If we excluded a physical partition, all its child elements would also be excluded as well, but `onSkipElement` would only be called for the partition element.

Reply:
Thats fair, we should probably document that behavior

Reply:
Are you against keeping it? Im personally in favor of adding it back and documenting the pitfall you mentioned. It could be useful sometime in the future

Reply:
I Brought back the `onSkipElement`